velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -1698,6 +1698,21 @@ TEST_F(StringFunctionsTest, toBase64Url) {
   EXPECT_EQ("_0-_UA==", toBase64Url(fromHex("FF4FBF50")));
 }
 
+TEST_F(StringFunctionsTest, toIEEE754_32) {
+  const auto toIEEE754_32 = [&](std::optional<float> value) {
+    return evaluateOnce<std::string, float>("to_ieee754_32(c0)", value);
+  };
+
+  EXPECT_EQ(std::nullopt, toIEEE754_32(std::nullopt));
```

Comment:
Please can you add more tests for very large and very small values.

Reply:
Absolutely, let me add more tests. Also, thanks for the heads-up on the upcoming split. I'll wait for split PR to get merged and then make changes in PR accordingly.

Reply:
@aditi-pandit I have added the smallest and largest float number testcases.